semantics/llvm-syntax.k

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2013-2014 K Team. All Rights Reserved.
+/*@
+
+This is the syntax of LLVM.
+
+This syntax requires a input LLVM program without any comments on it.
+*/
+module LLVM-SYNTAX
+
+    /*
+    Define rules for non-keyword tokens. This is currently just a bunch
+    of hacks. They don't cover many valid forms of tokens, and they also
```

Comment:
Why are these a "bunch of hacks"?  We do not want any hacks.  You language in this comment is not appealing to potential users of your semantics.  Think about them when you write comments.  Talk to Radu if needed.  And yes, these token definitions look quite ugly, you should do better than that.